### PR TITLE
Improve the introduction of CN dataset

### PR DESCRIPTION
**Pull request by seisman**
Merge the overview into the index page for dataset-CN.

### CODE REVIEW

**Review by CovMat — APPROVED** [review]

File: source/dataset-CN/index.rst [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
    CN-border
    CN-block
+
+**下载数据**
+
+- 项目主页： https://github.com/gmt-china/china-geospatial-data
+- 下载地址： https://github.com/gmt-china/china-geospatial-data/releases
+
+macOS/Linux 用户请下载数据 :file:`china-geospatial-data-UTF8.zip`\ （UTF8 编码），
+Windows 用户请下载数据 :file:`china-geospatial-data-GB2312.zip` （GB2312 编码）。
```
Comment (core-man) [inline source/dataset-CN/index.rst:22]: This looks better?
```suggestion
.. note::

   - macOS/Linux 用户请下载数据 :file:`china-geospatial-data-UTF8.zip`\ （UTF8 编码）
   - Windows 用户请下载数据 :file:`china-geospatial-data-GB2312.zip` （GB2312 编码）
```

File: source/dataset-CN/index.rst [hunk 2]
```diff
@@ -1,10 +1,29 @@
 中国地理空间数据集
 ##################
 
+“中国地理空间数据集”包含了一些中国区域的地理空间数据，可以直接用于GMT绘图。
+数据由 GMT 用户提供，并由 GMT 中文社区收集和维护。
+
+目前，数据集包含如下地理空间数据：
```
Comment (core-man) [inline source/dataset-CN/index.rst:7]:
```suggestion
**数据列表**
```

File: source/dataset-CN/index.rst (hunk 1 above)
Comment (seisman) [reply source/dataset-CN/index.rst:22]: **Old**
image

**New**
image

Comment (core-man) [reply source/dataset-CN/index.rst:22]: Looks better~

File: source/dataset-CN/index.rst [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
    CN-border
    CN-block
+
+**下载数据**
+
+- 项目主页： https://github.com/gmt-china/china-geospatial-data
+- 下载地址： https://github.com/gmt-china/china-geospatial-data/releases
+
+macOS/Linux 用户请下载数据 :file:`china-geospatial-data-UTF8.zip`\ （UTF8 编码），
+Windows 用户请下载数据 :file:`china-geospatial-data-GB2312.zip` （GB2312 编码）。
```
Comment (core-man) [inline source/dataset-CN/index.rst]: Or this style?
```suggestion
- 项目主页： https://github.com/gmt-china/china-geospatial-data
- 下载地址： https://github.com/gmt-china/china-geospatial-data/releases

  - macOS/Linux 用户请下载数据 :file:`china-geospatial-data-UTF8.zip`\ （UTF8 编码）
  - Windows 用户请下载数据 :file:`china-geospatial-data-GB2312.zip` （GB2312 编码）
```

Comment (core-man) [reply source/dataset-CN/index.rst]: Sorry that I cannot build GMT_Docs locally now.

**Review by core-man — APPROVED** [review]

File: source/dataset-CN/index.rst [hunk 4]
```diff
@@ -1,10 +1,31 @@
...
+
+**下载数据**
+
+- 项目主页： https://github.com/gmt-china/china-geospatial-data
+- 下载地址： https://github.com/gmt-china/china-geospatial-data/releases
+
+.. note::
+
+   - macOS/Linux 用户请下载数据 :file:`china-geospatial-data-UTF8.zip`\ （UTF8 编码）
+   - Windows 用户请下载数据 :file:`china-geospatial-data-GB2312.zip`\ （GB2312 编码）
```
Comment (core-man) [inline source/dataset-CN/index.rst:24]:
```suggestion
   .. note::

      - macOS/Linux 用户请下载数据 :file:`china-geospatial-data-UTF8.zip`\ （UTF8 编码）
      - Windows 用户请下载数据 :file:`china-geospatial-data-GB2312.zip`\ （GB2312 编码）
```

**Comment by core-man** [comment]
I‘m okay with the two styles above. Please choose either one you prefer.

File: source/dataset-CN/index.rst (hunk 4 above)
Comment (seisman) [reply source/dataset-CN/index.rst:24]: 为何要缩进

Comment (core-man) [reply source/dataset-CN/index.rst:24]: 因为这个 note 跟下载选项（`下载地址： https://github.com/gmt-china/china-geospatial-data/releases`）直接相关(belong to)，而不是并列guanxi。

Comment (core-man) [reply source/dataset-CN/index.rst:24]: My Chinese input method is something wrong. Users will have to read the note after they read the downloading link.

Comment (seisman) [reply source/dataset-CN/index.rst:24]: 我并不这样觉得，这个 note 是属于“下载数据”这一小节的，而不是属于下载地址那个链接。这也是我为何一开始用一段话而不是一个列表写的原因。

Comment (core-man) [reply source/dataset-CN/index.rst:24]: 我都可以其实，按你喜欢的方式即可

Comment (seisman) [reply source/dataset-CN/index.rst]: 改回去了

**Pull request merged**